backend/app/core/doctransform/service.py

```
@@ -0,0 +1,138 @@
+import tempfile
+import shutil
+import logging
+from pathlib import Path
+from uuid import uuid4, UUID
+
+from fastapi import BackgroundTasks, UploadFile
```

Ruff (0.12.2)

`fastapi.UploadFile` imported but unused

Remove unused import: `fastapi.UploadFile`

(F401)